include/sqlite_orm/sqlite_orm.h

```diff
@@ -5340,28 +5343,26 @@ namespace sqlite_orm {
                     using member_pointer_t = typename column_type::member_pointer_t;
                     using getter_type = typename column_type::getter_type;
                     using setter_type = typename column_type::setter_type;
-
-					// GCC has bug that reported in https://gcc.gnu.org/bugzilla/show_bug.cgi?id=64095 so static_if must have at lease one input
                     if(!res){
-                        static_if<std::is_same<C, member_pointer_t>{}>([&res, &obj, &col](const C& c){
```

Comment:
I didn't make any manual changes to the single-header `sqlite_orm.h` - I just ran the amalgamation script to update it from the changes in `dev/`. It seems like e422eb2976f101419299ae6146473e15bacf135c changed it here instead of in `dev/table.h` there. I'll update `table.h` to prevent this from happening again.

Reply:
Yes, it is exactly what I mean. Thank you